transcendence-app/src/user/infrastructure/db/user.entity.ts

```diff
@@ -10,30 +13,31 @@ export enum userKeys {
   AVATAR_Y = '"avatarY"',
 }
 
-type UserEntityData = {
+interface UserData {
   id: string;
   username: string;
   email: string;
   fullName: string;
   password: string | null;
-  avatarId: string | null;
+  avatarId: string;
   avatarX: number;
   avatarY: number;
   createdAt: Date;
-};
+}
 
-export class UserEntity {
+export class User {
```

Comment:
I think is a good idea to keep the `*Entity` denomination, this way it's clear is data coming from the db

Reply:
Discussed by call: could be but a lot of work changing things on frontend, so no worth it